core/src/main/java/org/apache/accumulo/core/metadata/schema/CompactionMetadata.java

```diff
@@ -96,7 +101,7 @@ public FateId getFateId() {
   // This class is used to serialize and deserialize this class using GSon. Any changes to this
   // class must consider persisted data.
   private static class GSonData {
-    List<String> inputs;
+    String inputs;
```

Comment:
Wonder if we could make this inputs field a JsonElement like this [SO post](https://stackoverflow.com/questions/34167410/how-to-ask-gson-to-avoid-escaping-json-in-a-json-response) describes to avoid base64 and avoid escaping.

Reply:
That could be an alternative, but instead of the json object example from the post (which looks like a map), we would need to use a json array of strings.